docs/operator/configuration.md

```diff
@@ -20,7 +20,7 @@ It defines default configuration options, like images for components, timeouts,
 In addition, the operator has a special startup mode for outputting all variables, their types and default values.
 For instance, with this mode you can know versions of VM components, which are used by default: 
 
-```sh
+```console
```

Comment:
@tenmozes , @hagen1778 , @Amper , PTAL. It looks like we need updating docs at [VictoriaMetrics operator](https://github.com/VictoriaMetrics/operator/tree/master/docs), so they become compliant with the new scheme after 6c6c2c185f2cdd183964d980c1ae62f85e79e0bd and 491287ed1552798161c6e197fbf391770602d2b9

Reply:
@valyala yeah, please ignore this, should be fixed in https://github.com/VictoriaMetrics/operator/pull/862.